hugo/layouts/partials/header.html

```diff
@@ -12,10 +12,32 @@
             <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
                 <ul class="navbar-nav">
                     <li class="nav-item">
-                        <a class="nav-link" href="engine">source{d} Engine</a>
+                        <a class="nav-link" href="why-sourced">Why source{d}</a>
```

Comment:
The `Why source{d}` and `Features` links have not visible hover state.

Normal state :point_down:
![image](https://user-images.githubusercontent.com/2437584/58350183-774f6780-7e65-11e9-9ac4-f6e671785b2a.png)

`Features` and `Resources` where hovered; `Features` almost didn't change :point_down:
![image](https://user-images.githubusercontent.com/2437584/58349919-e24c6e80-7e64-11e9-8397-6a203e8be1d7.png)